Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -925,9 +958,19 @@ - (void)_ensureBinaryCompatibilityWithAsyncInteractionsIfNeeded
     static std::once_flag onceFlag;
     std::call_once(onceFlag, ^{
 #if HAVE(UI_ASYNC_TEXT_INPUT)
-        if (_page->preferences().useAsyncUIKitInteractions())
+        if (_page->preferences().useAsyncUIKitInteractions()) {
             class_addProtocol(self.class, @protocol(UIAsyncTextInput));
-#endif
+            unsigned methodCount = 0;
+            using MethodDescriptionList = objc_method_description[];
+            auto methods = std::unique_ptr<MethodDescriptionList, WTF::SystemFree<MethodDescriptionList>> {
+                protocol_copyMethodDescriptionList(@protocol(UIAsyncTextInput), NO, YES, &methodCount)
+            };
+            for (unsigned i = 0; i < methodCount; i++) {
+                if (![self.class instancesRespondToSelector:methods.get()[i].name])
```

Comment:
Optional in general, but also because EWS is 🐌 so maybe don't waste cycles... `std::unique_ptr<T[]>` instantiations have [`T& operator[]`](https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_at) defined for this use case.
```suggestion
                if (![self.class instancesRespondToSelector:methods[i].name])
```